calcom/cal.com
# fix: lint fix for lint errors introduced in PR#10225

## What does this PR do?

Fixes lint errors introduced in PR#10225



## Requirement/Documentation


  - If there is a requirement document, please, share it here.
  - If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?



  - Are there environment variables that should be set?
  - What are the minimal test data to have?
  - What is expected (happy path) to have (input and output)?
  - Any other important info that could help to test that PR

The Github workflow check should be evidence enough

## Mandatory Tasks


## Checklist
